syntax = "proto3";

option optimize_for = SPEED;
option go_package = "github.com/dalmarcogd/mobstore/products/internal/domains/domainsgrpc;domainsgrpc";

package domains;

message DiscountRequest {
  string product_id = 1;
  string user_id = 2;
}

message DiscountResponse {
  string product_id = 1;
  string user_id = 2;
  float percentage = 3;
  float value_in_cents = 4;
}